cirq/linalg/decompositions_test.py

```diff
@@ -724,3 +724,16 @@ def test_kak_vector_negative_atol():
 def test_kak_vector_input_not_unitary():
     with pytest.raises(ValueError, match='must correspond to'):
         cirq.kak_vector(np.zeros((4, 4)))
+
+
+@pytest.mark.parametrize('unitary', [
+    cirq.testing.random_unitary(4),
```

Comment:
Optional: I wonder what distribution we get here on the elements of KAK (e.g. on the three interaction coefficients). A good addition to the random unitary test would be one synthesized from known coefficients, e.g.

```python
u = np.exp(1j*np.pi*(0.2*cirq.unitary(cirq.XX) + 0.3*cirq.unitary(cirq.YY) + 0.4*cirq.unitary(cirq.ZZ)))
```

Reply:
I added a couple more cases. The "arbitrary fixed coefficients" case is very well covered by the random unitary; the corner cases have to do with interaction coefficients being zero or maxed.